samples/photo_album/src/components/PhotoThumbnails.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {Image, Transformation} from 'cloudinary-react';
+
+const transformations = [
+    [{crop: 'fill', width: '150', height: '150', radius: '10'}],
+    [{crop: 'scale', width: '150', height: '150'}],
+    [{crop: 'fit', width: '150', height: '150'}],
+    [{crop: 'thumb', gravity: 'face', width: '150', height: '150'}],
+    [{crop: 'fill', effect: 'sepia', gravity: 'north', width: '150', height: '150', radius: '20'}, {angle: '20'}]
+];
+
+const PhotoThumbnails = ({publicId}) => (
+    <table className="thumbnails">
```

Comment:
Use CSS grid instead of table (good chance to try it out)

Reply:
Pass..